apps/app/.env.sample

```
@@ -9,6 +9,10 @@ NEXT_PUBLIC_BACKEND_URL=http://localhost:8080
 NEXT_PUBLIC_PANIER_URL=http://localhost:3002
 NEXT_PUBLIC_SITE_URL=http://localhost:3001
 NEXT_PUBLIC_ENV_NAME=dev
+# Content-Security-Policy : par défaut posée en `Report-Only` (collecte des
+# violations sans rien bloquer). Mettre `CSP_ENFORCE=true` pour passer en mode
+# bloquant une fois l'allowlist validée en préprod.
+CSP_ENFORCE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CSP_ENFORCE key should go before the NEXT_PUBLIC_AUTH_URL key

(UnorderedKey)